packages/lib/src/forms/validators.ts

```diff
@@ -80,6 +80,16 @@ export const validPercentage: Validator<number> = value => {
   return null;
 };
 
+export const validQueuedVotePercentage: Validator<number> = value => {
+  const error = "Percentages must be between 50 and 100.";
+
+  if (value > 100 || value < 50) {
```

Comment:
This logic can be achieved with 2 validation functions, `.validators(greaterThanOrEqual(50), lessThanOrEqual(100))`